mass_mailing_switzerland/models/mass_mailing_contact.py

```diff
@@ -220,7 +224,10 @@ def write(self, values):
 
     @api.multi
     def unlink(self):
-        self.action_archive_from_mailchimp()
+        try:
+            self.action_archive_from_mailchimp()
+        except Exception as e:
+            logger.error(traceback.format_exc())
```

Comment:
This way we don't need the `traceback` library.
```suggestion
        except:
            logger.error("Error archiving mailchimp contact", exc_info=True)
```